releasenotes/notes/chdir-to-writable-dir-for-in-toto-68a834c59e9986ff.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    For in-toto, run pip in a directory writable by the Agent, so that we can
```

Comment:
I think it's only `readable` instead of `writable` since the goal is just to allow switching back to this directory